rb/lib/selenium/webdriver/chrome/bridge.rb

```diff
@@ -26,15 +26,25 @@ def initialize(opts = {})
           opts[:desired_capabilities] = create_capabilities(opts)
 
           unless opts.key?(:url)
+            driver_path = opts.delete(:driver_path) || Chrome.driver_path
             port = opts.delete(:port) || Service::DEFAULT_PORT
-            service_args = opts.delete(:service_args) || {}
 
-            if opts[:service_log_path]
-              service_args.merge!(service_log_path: opts.delete(:service_log_path))
+            opts[:driver_opts] ||= {}
+            if opts.key? :service_log_path
+              warn <<-DEPRECATE.gsub(/\n +| {2,}/, ' ').freeze
```

Comment:
Heh, yes, I asked if that's the way we should be doing it in #3584. I only did it this way because that's how it's [currently being done](https://github.com/SeleniumHQ/selenium/search?l=Ruby&q=warn). So we should probably update all of these as well.